modules/services/agentless-scanning/main.tf

```diff
@@ -231,15 +243,25 @@ data "aws_iam_policy_document" "key_policy" {
 }
 
 resource "aws_kms_key" "agentless" {
+  count = var.deploy_global_resources ? 1 : 0
+
   description             = "Sysdig Agentless encryption key"
   deletion_window_in_days = var.kms_key_deletion_window
   key_usage               = "ENCRYPT_DECRYPT"
-  policy                  = data.aws_iam_policy_document.key_policy.json
+  policy                  = data.aws_iam_policy_document.key_policy[0].json
   multi_region            = true
   tags                    = var.tags
 }
 
-resource "aws_kms_alias" "agentless" {
-  name          = "alias/${var.name}"
-  target_key_id = aws_kms_key.agentless.key_id
+resource "aws_kms_replica_key" "agentless_replica" {
+  count = var.deploy_global_resources ? 0 : 1
+
+  description             = "Multi-Region replica key"
```

Comment:
Maybe add a `Sysdig` to this description?

Reply:
yes, makes sense. Updated and fixed in next commit change.